package.json

```diff
@@ -76,7 +76,7 @@
   "devDependencies": {
     "browserify": "^14.1.0",
     "cross-env": "^3.2.4",
-    "electron": "1.7.5",
+    "electron": "^1.8.1",
```

Comment:
looks good but i would used a fix version: 1.8.1

Reply:
Makes sense. I've pushed an update.

I tried building with node 6.10.0, didn't see any problems. I guess we should be good to go.